app/app.js

```diff
@@ -27,6 +27,7 @@ const mime = require('mime-types');
 const imgSize = require('image-size');
 const util = require('util');
 const log = require('./log');
+const apm = require('./apm');
```

Comment:
as for the linting error: we could remove your `apm.js` file and initialize in app.js so that it gets used.

```suggestion
const apm = require('elastic-apm-node');

apm.start({
  serviceName: process.env.ELASTIC_APM_SERVICE,
  secretToken: process.env.ELASTIC_APM_TOKEN,
  serverUrl: process.env.ELASTIC_APM_SERVER_URL,
  environment: process.env.STAGE,
});
```